### DISCUSSION TITLE
linux es mejor que windows?

### DISCUSSION THREAD

**Discussion by JamSwiftDev** [hackatomik/SISCOx256#4] [Encuesta | Carrera]
quiero ver sus votaciones y comentarios.

**Comment by JamSwiftDev** [comment]
pienso que no sinceramente a linux le faltan muchas cosas

**Reply by JamSwiftDev** [reply]
yo creo que no linux es mas personalizable y mas versatil

**Comment by tapilew** [comment]
Linux es de código abierto. Libertad absoluta.

**Comment by galib7u7** [comment]
Tiene muchisimas ventajas, además de que es gratis a comparación del windows que hay que comprar la licencia, ya si habalmos de jugar videojuegos pues bueno😭
